recipes/octokitpy-routes/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "octokitpy-routes" %}
+{% set version = "2.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 07f5b4acd1fe09da3c664062386fe7482ff34a9219737debec460632277616b3
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  # Currently can't import specifications on Windows
+  # Tracked under issue https://github.com/khornberg/octokitpy-routes/issues/8
```

Comment:
Does it make sense to remove the `noarch: python` and add a `skip: True  # [win]` for now until this is fixed?

Reply:
@synapticarbors on some previous recipes, reviewers have suggested as leaving it as noarch; that way, as soon as the issue is rectified it will work without an update required. happy to change it if that's your preference!

Reply:
that sounds reasonable to me.